Website/src/app/management/poi/client.tsx

```diff
@@ -172,157 +172,157 @@ export default function POIManagement({ poiTypes, tracks }: { poiTypes: POIType[
 			<form onSubmit={updatePoi} ref={formRef} className={"grid grid-cols-8 gap-2 mx-1.5 items-center"}>
 				{
 					/* Display a success message if the success flag is true */ success ? (
-						<SuccessMessage {...{ setSuccess, setModified }} />
```

Comment:
I think we should be more careful not to change the formatting too much in unrelated areas, or else tools like ´git blame´ might become useless.